splunknozzle/nozzle_test.go

```diff
@@ -127,6 +131,27 @@ var _ = Describe("SplunkFirehoseNozzle", func() {
 		Expect(f).ToNot(BeNil())
 	})
 
+	It("Montoring Enabled", func() {
+		Monitoring := noz.Metric()
+		var ok bool
+		if _, ok = Monitoring.(*monitoring.Metrics); ok {
+		}
+
+		Expect(ok).To(Equal(true))
+
+	})
+
+	It("Montoring Disabled", func() {
+		config.MetricMonitorInterval = 0 * time.Second
+		Monitoring := noz.Metric()
+		var ok bool
+		if _, ok = Monitoring.(*monitoring.Metrics); ok {
```

Comment:
No need to declare bool var.
Directly use `if _, ok := Monitoring.(*monitoring.Metrics); ok {`

Reply:
Updated in the latest commit